app/dashboard/models.py

```diff
@@ -987,6 +990,17 @@ def additional_funding_summary_sentence(self):
 
         return sentence
 
+    @property
+    def reserved_for_user_handle(self):
+        if self.bounty_reserved_for_user:
+            return self.bounty_reserved_for_user.handle
+        return None
+
+    @reserved_for_user_handle.setter
```

Comment:
😭

Reply:
@pinkiebell Can you eliminate the getter/setter here and replace it with a plain attribute unless you really need to use the setter/getter here?